config/file_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) OpenFaaS authors 2017. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package config
+
+import (
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+func Test_LookupAuthConfig_WithNoConfigFile(t *testing.T) {
+	DefaultFile = "test1.yml"
+	_, _, err := LookupAuthConfig("http://openfaas.test1")
+
```

Comment:
Can you get rid of the extra line between the function calls and error handling, same for the rest of this file.